bucket/espanso-alpha.json

```
@@ -0,0 +1,42 @@
+{
+    "version": "2.1.3-alpha",
+    "description": "Text expander",
+    "homepage": "https://espanso.org",
+    "license": "GPL-3.0-only",
+    "suggest": {
+        "Microsoft Visual C++ Redistributables 2019": "extras/vcredist2019"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/federico-terzi/espanso/releases/download/v2.1.3-alpha/Espanso-Win-Installer-x86_64.exe",
+            "hash": "5c3b5b4c14671d0fb56219499d12f10241680d3e996f92f84576274655b381f0"
+        }
+    },
+    "innosetup": true,
+    "bin": [
+        "espansod.exe",
+        "espanso.cmd"
+    ],
+    "shortcuts": [
+        [
+            "espansod.exe",
+            "Espanso",
+            "start",
```

```suggestion
            "launcher",
```

Shouldn't this be "launcher" instead?